ApiKeys/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "google/cloud-api-keys",
```

Comment:
Just a thought (and I know it's too late to change for this package now), but we have examples of when camelcase is not a hyphen but is a namespace (such as `google/cloud-bigquerydatatransfer`) and also cases where it is a hyphen but is camelcase, but not a namespace (like this  one). So it seems the hyphen in the package name is arbitrary.

I would advocate that we start using the hyphen to represent a namespace, rather than camelcase. This would bring consistency to our package names, but also be advantageous for having virtual packages sometimes in the future (for example, `google/cloud-bigtable-all`, `google/cloud-beyondcorp-all` and `google/cloud-iam-all`), where they all share a same namespace. If we go with this convention, then it's a bit strange for packages like `google/cloud-api-gateway` and `google/cloud-api-keys`, since they look like they're related but they aren't (and they don't share a namespace).

Curious your thoughts.

Reply:
Your proposal sounds reasonable to me 👍.